pkg/daemon/ceph/config/config.go

```diff
@@ -69,6 +69,7 @@ type GlobalConfig struct {
 	ClusterNetwork           string `ini:"cluster network,omitempty"`
 	MonKeyValueDb            string `ini:"mon keyvaluedb"`
 	MonAllowPoolDelete       bool   `ini:"mon_allow_pool_delete"`
+	MonAllowPoolSizeOne      bool   `ini:"mon_allow_pool_size_one"`
```

Comment:
We can add the new setting to the mon store instead of via ceph.conf, right? 

I thought the ceph.conf was only used for legacy OSDs, which are gone now. @BlaineEXE Can we remove this ceph.conf generation now?

Reply:
Not entirely. This is the file from the operator. We would put it in the mon/kv store but this doesn't mean that file won't exist anymore. They are fields that are still easier to manage like the `mon_host`. Also, setting something in the mon kv store is "expensive" since it requires a CLI call wherewith the ceph.conf it's already there.

Both options are valid, I just went for the ceph.conf one since we will likely never be able to remove it entirely without over-engineering.

Reply:
oh of course, we still need it in the operator. The daemons just don't use it.